tests/robustness/traffic/key_window.go

```diff
@@ -0,0 +1,62 @@
+package traffic
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+)
+
+type KeyWindow struct {
+	sync.Mutex
+
+	Counter    int
```

Comment:
if we don't use this anywhere, I think let's not make it public.

Reply:
All those fields should be private and we should fill them through constructor.